data/coalition/coalition missions.txt

```diff
@@ -3211,3 +3211,612 @@ mission "Heliarchs Buy Jump Drive Mission"
+
+mission "Coalition: Struggling Author 2"
+	name "Help Anna register with the Heliarchy"
+	description "Take Anna Fangvin to the <destination> so she can register with the Heliarchs."
+	source "Ki Patek Ka"
+	destination "Ring of Friendship"
+	passengers 1
+	to offer
+		has "Coalition: Struggling Author 1: done"
+	on offer
+		conversation
+			`You walk through the spaceport, looking for Anna. You find her in the middle of a public square, seated at a table with her laptop out. You figure she must've intended on getting started on some early work, what with the newly found inspiration of being in such an alien society, but it seems the sheer volume and diversity of peoples here on <origin> made her forget to type entirely.`
+			`	She only seems to notice you when you also sit down beside her. "Oh, hello again, <first>!" she says, snapping back into focus. "Is the ship ready?"`
+			`	Hearing an affirmative answer, Anna stands up from her chair and begins following you to the <ship>. She settles into her bunk as you prepare for the trip to the <destination>.`
+				accept
+	on enter "Quaru"
+		conversation
+			`As you enter <system>, Anna walks out of her bunk and finds a spot on the side of the bridge, peering into the empty void. "What a massive ringworld!" she exclaims.`
+			choice
+				`	"It's enormous. As you know, they stole it from the Quarg roughly seven thousand years ago."`
+				`	(Say nothing.)`
+			`	"This is going to be a great source of inspiration for my next novel," says Anna, still in awe. She pulls out a small notebook and begins busily writing in it.`
+	on complete
+		conversation
+			`Anna is amazed as you touch down on the ringworld. As the two of you open the hatch, you see a Heliarch agent in front of you.`
+			`	"Greetings, humans. For you, what can I do today?" asks the Heliarch. You respond that you would like to register Anna with the Heliarchy. After reaching the registration office, Anna enters it, coming out after fifteen minutes. "They finally got all the paperwork done. I'm off to the spaceport to get some food. Meet me there to discuss our next move." The Heliarch follows her to the cafeteria area.`
+
+
+mission "Coalition: Struggling Author 3"
+	name "Back to <planet>"
+	description "Now that Anna Fangvin has completed the registration process, take her and the Heliarch agent to get a translation device on <destination>."
+	source "Ring of Friendship"
+	destination "Ki Patek Ka"
+	passengers 2
+	blocked "You do not have enough space for Anna and the Heliarch agent. Return here when you have two bunks free."
+	to offer
+		has "Coalition: Struggling Author 2: done"
+	on offer
+		conversation
+			`Going through the spaceport looking for Anna, you finally find her at a restaurant, eating a longcow steak and typing something on her laptop. Beside her sits a Heliarch Saryd, who points at something on the screen. She sees you and motions for you to take a seat.`
```

Comment:
Horse (and I presume Centaurs/Saryds?) don't have the anatomy to sit naturally without great discomfort and possible damage - they're much more comfortable standing. Tweaked for the Heliarch to be standing instead.
```suggestion
			`Going through the spaceport looking for Anna, you finally find her at a restaurant, eating a longcow steak and typing something on her laptop. The Heliarch Saryd from earlier stands beside her, bent-over and gesturing at something on the screen. She sees you and motions for you to take a seat.`
```